packages/nova-react-test-utils/src/shared/storybook-nova-decorator-shared.tsx

```diff
@@ -82,7 +83,11 @@ export function getRenderer(
 ): React.FC<React.PropsWithChildren<unknown>> {
   if (query) {
     const Renderer: React.FC<unknown> = () => {
-      const { data } = useLazyLoadQuery(query, variables);
+      const { data } = useLazyLoadQuery(
+        // There are no consequences of the cast, we do it only to make sure pure relay components can also leverage the decorator
```

Comment:
During runtime this would be replaced with the runtime document, so it's not actually even this type.

Which when using Nova and Relay should be the same document. But not sure if that is worth adding to the comment?